Registry.toml

```diff
@@ -27,6 +27,7 @@ cd197ee6-b873-11e8-324e-2bd8ebd9d51c = { name = "BlinkPOMDPSimulator", path = "B
 cef570c6-3a94-5604-96b7-1a5e143043f2 = { name = "SARSOP", path = "S/SARSOP" }
 d721219e-3fc6-5570-a8ef-e5402f47c49e = { name = "BasicPOMCP", path = "B/BasicPOMCP" }
 d96c9ae4-3372-47d5-8a88-316ae77be8cf = { name = "ARDESPOT", path = "A/ARDESPOT" }
+de008ff0-c357-11e8-3329-7fe746fe836e = { name = "RockSample", path = "R\\RockSample" }
```

Comment:
Umm. This will be an issue on *nixes.

Reply:
what are *nixes? 
I tried a new registration script btw: https://github.com/JuliaRegistries/Registrator.jl/issues/197#issuecomment-517812396

Reply:
Unix-like operating systems like Linux and Mac osx. They use forward instead of back slashes like windows. Forward slashes between files are more standard. I wonder if Julia will just handle it though

Reply:
Aah I see, I guess it might come from the fact that I ran this script from my local windows machine.
Running `] add RockSample` from a linux machine seems to work without any issues.